o-clock.cabal

```diff
@@ -14,10 +14,10 @@ build-type:          Simple
 extra-doc-files:     CHANGELOG.md
```

Comment:
Could you also please add `stability: stable` to these fields?

* https://www.haskell.org/cabal/users-guide/developing-packages.html#pkg-field-stability

I think it can be slightly better to show that library is stable than to not show it.